packages/block-library/src/site-logo/edit.js

```diff
@@ -192,6 +196,24 @@ const SiteLogo = ( {
 						value={ width || '' }
 						disabled={ ! isResizable }
 					/>
+					<ToggleControl
+						label={ __( 'Make image a link to home' ) }
```

Comment:
```suggestion
						label={ __( 'Add link to home' ) }
```

Reply:
`Add link to home` would imply to some that a link will be added. As in, add a separate link - as opposed to convert the image to a link. Maybe `Link image to home` would be clearer?